templates/windows.tpl.yaml

```diff
@@ -63,9 +59,6 @@ metadata:
       ]
 
   labels:
-    os.template.kubevirt.io/win2k19: "true"
-    os.template.kubevirt.io/win2k16: "true"
-    os.template.kubevirt.io/win2k12r2: "true"
     workload.template.kubevirt.io/{{ item.workload }}: "true"
```

Comment:
Remove `workload` and `flavor` labels too please

Reply:
Done